.coveragerc

```diff
@@ -1,3 +1,3 @@
 [run]
 source = pynamodb/.
-omit = pynamodb/tests/integration/*
\ No newline at end of file
+omit = pynamodb/tests/*
```

Comment:
wait, what? :)

Reply:
Something with the merge commit from before was not visualizing correctly here.  I did a rebase instead now and things are much cleaner.